app/controllers/carts_controller.rb

```diff
@@ -1,8 +1,28 @@
 class CartsController < ApplicationController
+
+  def index
+    @orders = Order.all
+  end
+
   def show
     @order_items = current_order.order_items
+    @products = Product.all
   end
   
-  def myorder
+  def myorder; end
+
+  def place_order
+    if current_order.sent == true
```

Comment:
no need to check if a boolean is true or false, if `current_order.sent` is true then don't need to check if it's true, you can just do `if current_order.sent`

also, `sent` is not a good boolean variable name, please use the rails conventions `is` `has` `are` etc. in this case it should be `is_sent`